src/main/kotlin/com/openlattice/shuttle/Shuttle.kt

```diff
@@ -526,6 +526,7 @@ class Shuttle (
             val flightNames = flightPlan.keys.joinToString(", ") { it.name }
             val exceptionLog = "Encountered exception $ex while integrating flight plan containing flight(s) $flightNames"
             writeLog(flightNames, setOf(exceptionLog, ex.stackTrace.toString()), IntegrationStatus.FAILED)
+            throw java.lang.Exception(exceptionLog)
```

Comment:
I don't understand why this is necessary when we already log the exception and stacktrace on the line above

Reply:
An error in getting the payload results in a "SUCCESS" banner...

Reply:
https://jira.openlattice.com/browse/LATTICE-2404

Reply:
Also, we never check logs of each job (there's 1000's a day), only of the failed ones...

Reply:
I think that is an issue with the structure of MissionControl and not with shuttle though

Reply:
Still, errors should propagate and result in a failure, so wherever the error comes from, it should not be caught and logged, but thrown.

Reply:
I think the correct fix here is to call `MissionControl.fail()` instead of throwing a new exception

Reply:
Could also just remove the whole `catch()` statement?

Reply:
Right but then we would have to figure out what kind of error happened one or two layers above when we already have all the information we need in the `catch()` to write an informative log statement

Reply:
Actually, launch() will not throw an error, thus MissionControl will not catch it.
This fixes that, and MissionControl cleanly exists with a failure.

Reply:
So the problem here is

- if an exception is thrown here, shuttle server will break (which is why the exception is being caught)
- if an exception is *not* thrown here, the integrations not using shuttle server will be marked as successful if they fail.

So, we should either call `MissionControl.fail()` if and only if shuttle is being run *without* shuttle server, or let the integrations be marked as successful for extra motivation to switch over to shuttle server :)

Reply:
This situation happens: eg. a column just isn't present anymore in a recurring table.

The logs are informative, but we need to know to look at the logs.
Once we know something's up, we can dig deeper, but as long as we don't know, we just happily look at all the green checkmarks in rundeck.

This error message about a missing column is also enough for us to: (1) either change our flights or (2) ask customer why the column is gone.

Reply:
> Right but then we would have to figure out what kind of error happened one or two layers above when we already have all the information we need in the `catch()` to write an informative log statement

We can't mission control fail because that will exit the process and break shuttle server

Reply:
Piper pushed a fix for shuttle server breaking.

This will lead to data loss, so it's not just a matter of motivation.

Reply:
I think putting the mission control fail inside the if statement i just added sounds fine for now. It is what we do in other catches within shuttle.